rust/processor/src/processors/default_processor.rs

```diff
@@ -80,53 +92,62 @@ async fn insert_to_db(
         conn.clone(),
         insert_transactions_query,
         txns,
-        get_config_table_chunk_size::<TransactionModel>("transactions", per_table_chunk_sizes),
+        get_config_table_chunk_size::<TransactionModel>(
+            TransactionModel::table_name(),
+            per_table_chunk_sizes,
+        ),
     );
+
     let bmt_res = execute_in_chunks(
         conn.clone(),
         insert_block_metadata_transactions_query,
         block_metadata_transactions,
         get_config_table_chunk_size::<BlockMetadataTransactionModel>(
-            "block_metadata_transactions",
+            BlockMetadataTransaction::table_name(),
```

Comment:
I think @CapCap was looking into a way to automatically get the table name from diesel type, so we wouldn't have to do something like this...